src/tests/ExportNameRuleTests.ts

```diff
@@ -315,6 +315,24 @@ describe('exportNameRule', () : void => {
             ]);
         });
 
+        it('when mis-named function is exported in a separate statement', () : void => {
```

Comment:
Good test for the case of complaining. There should also be a corresponding test for the case of the user getting it right, e.g. `file.ts` having `export { file };`.

Reply:
Added in new commit, good catch!